packages/kbn-test/src/functional_test_runner/lib/config/run_check_ftr_configs_cli.ts

```diff
@@ -23,6 +23,14 @@ const THIS_REL = Path.relative(REPO_ROOT, THIS_PATH);
 const IGNORED_PATHS = [
   THIS_PATH,
   Path.resolve(REPO_ROOT, 'packages/kbn-test/src/jest/run_check_jest_configs_cli.ts'),
+  Path.resolve(
+    REPO_ROOT,
+    'x-pack/test/security_solution_api_integration/test_suites/security_solution_endpoint/configs/config.base.ts'
+  ),
+  Path.resolve(
+    REPO_ROOT,
+    'x-pack/test/security_solution_api_integration/test_suites/security_solution_endpoint_api_int/configs/config.base.ts'
+  ),
```

Comment:
Can you please help me understand why the base configs are added here? We usually put ignored base config files into the `disabled` section of `.buildkite/ftr_configs.yml`, see https://github.com/elastic/kibana/blob/main/.buildkite/ftr_configs.yml#L5 .

Reply:
My bad, I'll move them over disabled section of .buildkite/ftr_configs.yml